setup.py

```diff
@@ -59,4 +59,9 @@
             "Programming Language :: Python :: 3.8",
             "Topic :: Scientific/Engineering",
         ],
+        entry_points={
+            "prodigy_recipes": [
+                "pyannote.sad.manual = pyannote.audio.interactive.v03:sad_manual",
```

Comment:
Let's rename this recipe to `pyannote.voice_activity_detection`and rename `v03.py` to `voice_activity_detection.py`.